spec/components/previews/support_interface/feature_audit_trail_component_preview.rb

```diff
@@ -1,9 +0,0 @@
-module SupportInterface
-  class FeatureAuditTrailComponentPreview < ViewComponent::Preview
-    def pilot_open
-      render SupportInterface::FeatureAuditTrailComponent.new(
-        feature: Feature.find_by(name: 'pilot_open'),
-      )
-    end
-  end
-end
```

Comment:
I think this file should stay but with a different example feature flag.

Reply:
Good spot! Done ♻️